Ladybird/cmake/InstallRules.cmake

```diff
@@ -30,8 +40,9 @@ install(TARGETS WebContent
 
 include("${SERENITY_SOURCE_DIR}/Meta/Lagom/get_linked_lagom_libraries.cmake")
 get_linked_lagom_libraries(ladybird ladybird_lagom_libraries)
+get_linked_lagom_libraries(SQLServer webcontent_lagom_libraries)
```

Comment:
How was this tested? This should be `sqlserver_lagom_libraries`.

Reply:
Good catch, will fix. As for testing: I built this locally on Fedora 37, and then built it inside a mock container as part of an rpm package build and installed the result -- it worked fine. For context, I was trying to see how feasible it'd be to package ladybird for Fedora.

Reply:
Should be fixed now, and I did another build and retested it for good measure.